addons/account/controllers/download_edi_docs.py

```diff
@@ -5,7 +5,7 @@
 
 
 class EdiDocumentDownloadController(http.Controller):
-    @http.route('/account_edi/download_edi_documents', type='http', auth='user')
+    @http.route('/account/download_edi_documents', type='http', auth='user')
```

Comment:
You need to keep the original code in account_edi (stable policy) but it's ok to add a new copy-pasted code in account and make account_edi controller calling the one in account